src/test/java/org/cactoos/iterable/JoinedTest.java

```diff
@@ -75,25 +73,26 @@ public void joinsIterablesWithSize() {
             new IterableOf<>(1, -1, 0, 0),
             new IterableOf<>(1)
         );
-        MatcherAssert.assertThat(
-            "Can't concatenate four iterables together",
-            list, Matchers.iterableWithSize(10)
-        );
-        MatcherAssert.assertThat(
-            "Can't concatenate four iterables together, again",
-            list, Matchers.iterableWithSize(10)
-        );
+        new Assertion<>(
```

Comment:
@victornoel I don't understand the necessity of this test. I know that you just refactored the previous existing tese, but don't you think that we can remove this tes since we already are doing this checks on `transformsList` and `joinItemAndIterable`?

Reply:
@paulodamaso totally